.github/workflows/build_test_deploy.yml

```diff
@@ -49,6 +49,8 @@ jobs:
           dotnet user-secrets set "trams:api_key" "${{ secrets.TRAMS_API_KEY }}"
           dotnet user-secrets set "app:username" "${{ secrets.APP_USERNAME }}"
           dotnet user-secrets set "app:password" "${{ secrets.APP_PASSWORD }}"
+          dotnet user-secrets set "app:password" "${{ secrets.APP_PASSWORD }}"
+          dotnet user-secrets set "AzureAd:ClientSecret" "${{ secrets.AZURE_AD_CLIENT_SECRET }}"
```

Comment:
Apologies, I am going to reiterate that I don't think we should be using user-secrets in anything other than development as they are stored in plain text and are explicitly not designed for prod user - especially something that gives access to our AD.